src/assets/style/main.scss

```diff
@@ -0,0 +1,6 @@
+$green_color:#008000;
```

Comment:
For css/scss variables kindly use BEM guidelines, have a look at this [tuitorial](https://assist-software.net/blog/css-guideline-tutorial-bem-sass) and [example](https://github.com/jalyna/bem-scss-styleguide).

Reply:
Let me look into it